.eslintrc.js

```diff
@@ -0,0 +1,205 @@
+module.exports = {
+    env: {
+        es6: true,
+        node: true,
+    },
+    extends: [],
```

Comment:
😢 https://github.com/typescript-eslint/tslint-to-eslint-config/issues/232 is really hitting hard here. I was hoping this would extend these, similar to https://github.com/typescript-eslint/tslint-to-eslint-config/blob/4a88942110f47772736801e80ecb5fc70b62a2eb/.eslintrc.js:

```suggestion
    extends: ["plugin:@typescript-eslint/all", "prettier", "prettier/@typescript-eslint"],
```

...so any rules already enabled in `/all` won't need to be explicitly mentioned here. If there are recommended `extends` configs in the community rulesets you also added, those are great to add here too.